src/Core/Engine/Physics/Misc/PhysicsBody.hpp

```
@@ -0,0 +1,16 @@
+#include "Engine/Components/Physics/Collider.hpp"
+#include "Engine/Components/Physics/Rigidbody.hpp"
+#include "vendor/entt.hpp"
+
+struct PhysicsBody
+{
+    entt::entity entity{entt::null};
+    COMPRigidBody RigidBody;
+    COMPTransform Transform;
+    COMPCollider Collider;
+
+    PhysicsBody(entt::entity entity, COMPRigidBody& body, COMPTransform& transform, COMPCollider& collider)
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: RigidBody, Transform, Collider

(cppcoreguidelines-pro-type-member-init)